packages/flutter_localizations/test/basics_test.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('$MaterialLocalizations localizes text inside the tree', (WidgetTester tester) async {
```

Comment:
`$GlobalMaterialLocalizations`?

Reply:
Good catch, the name wasn't even close (not sure what the $ was doing there either).

This test checks nested Localizations.

Reply:
The `$` makes sure that if you type `$BlobalBacterialVocalizations` you get an error from the analyzer.